content/doc/developer/forms/form-validation.adoc

```diff
@@ -45,6 +45,21 @@ public FormValidation doCheckThreads(@QueryParameter("value") final String meani
 
 This naming convention is how Jenkins wires up your form validation method. The return type also must be FormValidation, which offers various static factory methods to create an instance. Form validation can be of 3 different kinds: `ok`, `warning` or `error`. Validation message can either be represented in plaintext or use HTML markup. Several validations of different kinds can be reported wrapping then into one using aggregate factory method.
 
+=== Setting up form validation within a Jelly template
+
+To configure form validation within a Jelly template to call `doCheckXxx` methods, defined in a particular class in Jenkins, you need to utilize the `checkUrl` and `checkDependsOn` attributes.
+Here's a breakdown of how these attributes are used:
+
+The `checkUrl` attribute specifies the URL that Jenkins requests to perform form validation.
+Typically, this URL points to the Jenkins server where it handles the validation request.
+The value entered in the input field is checked via AJAX against this URL and any errors encountered are rendered under the text field.
+This URL is associated with the method responsible for performing the validation (`doCheckXxx` method) within your Descriptor class.
+
+The `checkDependsOn` attribute defines the dependency between the field being validated and the field(s) whose values are required for validation.
+It specifies the ID(s) of the field(s) that the validation depends on.
```

Comment:
I don't think this is being suppored yet... It has to be a query parameter called `value` but not some query parameter name set by the developer.

Reply:
Maybe only `name` and `value` are supprted fields now for `checkDependsOn `?

Reply:
Yeah, I guess the wording as a whole just does not sit well with me.